### DISCUSSION TITLE
Update check not recognizing pretix 4.15

### DISCUSSION THREAD

**Discussion by fabm3n** [pretix/pretix#2949] [Q&A] [answered]
Hi,
my pretix 4.14 instance in not recognizing the new 4.15 update:
![image](https://user-images.githubusercontent.com/11352551/205690694-bc9b10b9-7e82-4b5d-b33f-bd9b9d1f5867.png)

The pages update is recognized.
Is this a similar issue like #2156?

Also, the PayPal Plugin is displayed twice?

Best regards
Fabian

**Comment by raphaelm** [comment, marked as answer]
>  my pretix 4.14 instance in not recognizing the new 4.15 update:

Sorry, our mistake. We're… not that good at manually changing that file as it looks. We'll automate that process at some point.

> Also, the PayPal Plugin is displayed twice?

Yupp, we actually do have two PayPal plugins in the system currently, one for the old PayPal API, one for the new one. In most places, this is invisible because one of them is always hidden. Here, it shows :shrug:
